sdk/eventhub/Azure.ResourceManager.EventHubs/CHANGELOG.md

```diff
@@ -101,10 +101,10 @@ var createEventHubResponse = this.EventHubManagementClient.EventHubs.CreateOrUpd
 
 After upgrade:
 ```C# Snippet:ChangeLog_Sample
-using Azure.Identity;
-using Azure.ResourceManager.Resources;
-using Azure.ResourceManager.EventHubs.Models;
-using Azure.Core;
+// using Azure.Identity;
```

Comment:
Why are these commented out?

Reply:
I think I understand - it's because we can't prevent building this using the ifdef SNIPPET anymore. An alternative would be to create a separate region around where the usings are declared in the cs file.

Reply:
Because they're not compilable in the location they were included - neither for the tests nor for customers doing a copy/paste on the snippet.   The most common approach that I've seen for including "using" hints in samples across MS Docs and elsewhere is to illustrate via comment, as I've done here.  

The other possibility that I see would be to create a separate snippet that demonstrates only the using statements, but that looked strange when I prototyped it, and I don't think the lack of locality was helpful in keeping the association of "using" context with the snippet; the comment does that better.